packages/server-wallet/src/protocols/__test__/direct-funding.test.ts

```diff
@@ -2,14 +2,14 @@ import {simpleEthAllocation, makeDestination, BN} from '@statechannels/wallet-co
 import matchers from '@pacote/jest-either';
```

Comment:
This file should be renamed?

Reply:
Oops, accidentally added a duplicate test file somehow when renaming